docs/managing-data-quality/snowplow-event-recovery/running/flink/index.md

```diff
@@ -16,15 +16,31 @@ sbt flink/assembly
 
 #### Running
 
-Using flink CLI:
+Event recovery jobs are usually ran using our [dataflow-runner](https://github.com/snowplow/dataflow-runner) application. An example configuration for running is [available in the repository](https://github.com/snowplow-incubator/snowplow-event-recovery/blob/master/.dataflow-runner).
```

Comment:
```suggestion
Event recovery jobs are usually run using our [dataflow-runner](https://github.com/snowplow/dataflow-runner) application. An example configuration for running is [available in the repository](https://github.com/snowplow-incubator/snowplow-event-recovery/blob/master/.dataflow-runner).
```

Reply:
That's passive voice, so I suppose the correct form is `ran`